pandas/util/_test_decorators.py

```diff
@@ -57,7 +57,11 @@ def safe_import(mod_name, min_version=None):
         return mod
     else:
         import sys
-        version = getattr(sys.modules[mod_name], '__version__')
+        try:
+            version = getattr(sys.modules[mod_name], '__version__')
+        except AttributeError:
```

Comment:
ugg, really?

Reply:
I suppose another way to go about this would be do a `str.lower` on the attribute, but I did it this way because it's clearer why there's even a need for it. Whenever xlrd becomes obsolete it's an easy identifier that we can get rid of it. 

If you prefer the former then no problem I'll do that in next commit

Reply:
FYI I put in a PR to xlrd to add this python-excel/xlrd#275 for future versions